network/anchor-platform/admin-guide/sep10/README.mdx

```diff
@@ -53,36 +59,63 @@ clients:
   #   - type: (required) `custodial` or `noncustodial`
   #
   #   If the type is `custodial`,
-  #   - signing_key: (required) the custodial SEP-10 signing key of the client.
+  #   - signing_keys: (required) the custodial SEP-10 signing key of the client. In the existence of signing_keys, the signing_key field is ignored.
+  #   - signing_key: (optional) DEPRECATED in favour of signing_keys. the custodial SEP-10 signing key of the client.
   #   - callback_url: (optional) the URL of the client's callback API endpoint.
   #   - allow_any_destination: (optional) default to false. If set to true, allows any destination for deposits.
   #   - destination_accounts: (optional) list of accounts allowed to be used for the deposit. If allows_any_destinations set to true, this configuration option is ignored.
   #
   #   If the type is `noncustodial`,
-  #   - domain: (required) the domain of the client.
+  #   - domains: (required) the domains of the client.
+  #   - domain: (optional) DEPRECATED in favour of domains. the domain of the client. In the existence of domains, the domain field is ignored.
```

Comment:
I think we should not reference deprecated fields, it's ok to just not include them in this documentation (but still have it in the yaml file)